ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -1014,17 +1051,20 @@ void handleBody(InputStream payload, MessageMetadata messageMetadata, MessageInf
         BlobData blobData = MessageFormatRecord.deserializeBlob(payload);
         ByteBuffer encryptionKey = messageMetadata == null ? null : messageMetadata.getEncryptionKey();
         ByteBuf chunkBuf = blobData.content();
+        this.isChunkCompressed = blobData.isCompressed();
```

Comment:
this is not used in this method

Reply:
Correct.  This is not used in this method.  It is used when generating the PutRequest.